frontend/taipy-gui/src/components/Taipy/TableFilter.tsx

```diff
@@ -101,20 +101,22 @@ const getActionsByType = (colType?: string) =>
 const getFilterDesc = (columns: Record<string, ColumnDesc>, colId?: string, act?: string, val?: string) => {
     if (colId && act && val !== undefined) {
         const colType = getTypeFromDf(columns[colId].type);
-        if (!val && (colType === "date" || colType === "number" || colType === "boolean")) {
+        if (val === "" && (colType === "date" || colType === "number" || colType === "boolean")) {
             return;
         }
         try {
             return {
                 col: columns[colId].dfid,
                 action: act,
                 value:
-                    colType === "number"
-                        ? parseFloat(val)
-                        : colType === "boolean"
-                        ? val === "1"
-                        : colType === "date"
-                        ? getDateTime(val)
+                    typeof val === "string"
```

Comment:
Unreadable for me... may be parenthesis and/or better indentation?

Reply:
i've added parenthesis and indentation but the formatter/linter changed it to this !